contracts/script/deploy/verifiable/DeployVerifiable.s.sol

```diff
@@ -116,24 +95,49 @@ contract DeployVerifiable is Script {
         // Transfer ownership of proxy admin to deployment initializer
         proxyAdmin.transferOwnership(address(deploymentInitializer));
 
+        _logs();
+
         vm.stopBroadcast();
 
-        _doTests(configData);
+        _doTests();
+    }
+
+    /// @dev override this if you don't want to use the environment to get the config path
+    function _initConfig() internal virtual {
+        cfg = vm.readFile(vm.envString("DEPLOY_CONFIG_PATH"));
     }
 
-    function _doTests(string memory cfg) internal {
-        vm.startPrank(initialOwner);
-        CalldataInitParams memory params = CalldataInitParamsLib.getCalldataInitParams(cfg);
+    function _logs() internal virtual {
+        console2.log("Deployment addresses:");
+        console2.log("Deployment Initializer: ", address(deploymentInitializer));
+        console2.log("Empty Contract Implementation", emptyContract);
+        console2.log("Mock Stake Registry Implementation", mockStakeRegistry);
+        console2.log("Mock Registry Coordinator Implementation", mockRegistryCoordinator);
+
+        console2.log(
+            "\n\nAll other relevant deployment addresses should be queried from the DeploymentInitializer contract."
+        );
+    }
+
+    function _doTests() internal {
```

Comment:
is this not a redundant simulation with just running the script with `--broadcast` enabled?

Reply:
ah I see that makes sense - any chance we could capture this role differentiation in a smol comment? At first glance the behavior is a bit hard to decipher

Reply:
https://github.com/Layr-Labs/eigenda/pull/1415/commits/acf9096d69f2e4f805e50c023de91269b4cde81b